parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/checker/BaseWorkFlowCheckerTask.java

```diff
@@ -75,7 +75,7 @@ public WorkReport execute(WorkContext workContext) {
 		if (escalationWorkflow != null && report.getStatus() == WorkStatus.FAILED
 				&& new Date().getTime() > expectedCompletionDate) {
 			// run escalation if SLA is exceeded
-			WorkFlowEngineBuilder.aNewWorkFlowEngine().build().run(escalationWorkflow, workContext);
+			return WorkFlowEngineBuilder.aNewWorkFlowEngine().build().run(escalationWorkflow, workContext);
```

Comment:
this is the engine to run escalation.  the returned report is not checker's report

Reply:
ah, so where is the return value captured from the escalation run?